imfilebrowser.h

```diff
@@ -646,6 +646,20 @@ inline void ImGui::FileBrowser::Display()
                     CloseCurrentPopup();
                 }
             }
+            else if (IsKeyPressed(ImGuiKey_GamepadFaceDown) && IsItemHovered()) {
```

Comment:
The spacing between 'if' and '(' seems inconsistent with the rest of the code, as does the missing linebreak before '{'.